src/pages/Hosts.tsx

```diff
@@ -60,7 +61,7 @@ const Hosts: React.FC = () => {
   }, [searchChanged, search, refetch]);
 
   const hosts = hostsData?.hosts;
-  const hostItems = hosts?.hosts ?? [];
+  const hostItems = (hosts?.hosts as Host[]) ?? [];
```

Comment:
Needed to do a type assertion here. Im not sure why it was needed tho since typescript should have properly asserted the correct type. cc. @tgrander  @SupaJoon

Reply:
Figured this out. The hostsTable component was expecting a prop of type `Host[]` but the Hosts query didnt return data that matched the Host Type. I updated the Hosts query to add 2 fields so that it more closely matches the Host type.